README.md

```diff
@@ -41,7 +41,7 @@ npm install --save @codemirror/basic-setup
 
 You can try out the latest release version of this mode on:
```

Comment:
This isn't true anymore because it's deployed from master and not the latest release.

Reply:
thanks for the review, I updated this line :). Will merge the PR.